python/tvm/relax/transform/transform.py

```diff
@@ -647,19 +647,21 @@ class FusionPattern(Object):
     pattern: DFPattern
     annotation_patterns: Mapping[str, DFPattern]
     check: Callable[[PatternCheckContext], bool]
+    attrs_getter: Callable[[PatternCheckContext], Dict[str, str]]
 
     def __init__(
         self,
         name: str,
         pattern: DFPattern,
         annotation_patterns: Optional[Mapping[str, DFPattern]] = None,
         check: Optional[Callable[[Mapping[str, Expr]], bool]] = None,
+        attrs_getter: Optional[Callable[[Mapping[str, Expr]], Dict[str, str]]] = None,
```

Comment:
Please use the same type annotation.

Reply:
Do you mean like: attrs_getter: Optional[Callable[[Mapping[str, Expr]], bool]] = None ? But the attrs_getter return a map of <str, str>, each element contains a attribute for the function. The attrs_getter and check has same argument, thus PatternCheckContext, but different return type.

Reply:
I mean, I was confused about the type annot difference between L650 and L658.

Reply:
I just copied from L649 and L657 and change return type ... from the check. I also don't know why different annotation is used for check 🤣...

Reply:
I see, let's fix them both.